data_format.go

```diff
@@ -30,9 +30,9 @@ const (
 	// XMLDataFormat specifies eXtensible Markup Language-formatted data
 	// currently not supported.
 	XMLDataFormat
-	// XLSDataFormat specifies microsoft excel formatted data
+	// XLSXDataFormat specifies microsoft excel formatted data
 	// currently not supported.
```

Comment:
Remove "currently not supported". I understand this is only partial support, but I think it's enough that it moves beyond "not supported".